package.json

```diff
@@ -65,7 +66,7 @@
     "karma-commonjs": "0.0.8",
     "karma-coverage": "0.2.6",
     "karma-growl-reporter": "0.1.1",
-    "karma-jasmine": "^0.1.0",
+    "karma-jasmine": "^0.1.6",
```

Comment:
Could we (and should we) move the test-related dependencies under `devDependencies`?

Reply:
Normally, that's exactly where these sorts of packages should go. I thought we had `devDependencies` at one point but we might have gotten rid of them. 

@adamayres do you have an objection if I move the karma pkgs to `devDependencies`?

Reply:
No objections.